toolchain/sem_ir/inst_namer.cpp

```diff
@@ -135,13 +135,10 @@ auto InstNamer::GetScopeIdOffset(ScopeIdTypeEnum id_enum) const -> int {
       offset += sem_ir_->classes().size();
       [[fallthrough]];
     case ScopeIdTypeEnum::For<ClassId>:
-      offset += sem_ir_->vtables().size();
-      [[fallthrough]];
-    case ScopeIdTypeEnum::For<VtableId>:
-      offset += sem_ir_->functions().size();
+      offset += sem_ir_->cpp_overload_sets().size();
       [[fallthrough]];
     case ScopeIdTypeEnum::For<CppOverloadSetId>:
-      offset += sem_ir_->cpp_overload_sets().size();
+      offset += sem_ir_->functions().size();
```

Comment:
This also looks off to me, indices after (things listed before in the switch) the CppOverloadSetIds aren't being offset by the size of cpp overload sets?